components/service/glean/metrics.yaml

```diff
@@ -102,6 +102,21 @@ glean.baseline:
     notification_emails:
       - telemetry-client-dev@mozilla.com
 
+  locale:
+    type: string
+    lifetime: application
+    send_in_pings:
+    - baseline
```

Comment:
Let's indent these as per the rest of the file.  (YAML doesn't require it, but I think it's clearer).

Reply:
Gah, I hate it when Android Studio kills indentation when copy pasting.